README.md

```diff
@@ -300,3 +300,9 @@ Solution: install cortx-py-utils RPM and retry.
 
 * [Hare RFCs](rfc/README.md)
 * [Hare wiki](https://github.com/Seagate/cortx-hare/wiki)
+
+## Tested by:
```

Comment:
Is this a requirement to add this particular section?
If it is a requirement, then is it a good idea to maintain it in a separate file named something like "Testing_Verification_History", as not to mix it up with contents of Readme? As this verification needs to be done at the end of every PI, the Readme will keep growing.

Reply:
Kishore (motr team) suggested to add this as a practice. I am ok to created separate page and link it to README 
https://github.com/Seagate/cortx-motr/blob/main/doc/Quick-Start-Guide.rst

Reply:
Created new Page.